plugins/org.eclipse.reddeer.e4.workbench/src/org/eclipse/reddeer/workbench/impl/part/AbstractWorkbenchPart.java

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc and others.
```

Comment:
There should be 2018

Reply:
actually the files were created in 2017 and there is no need to change the date for the updated files. A date in license header represents a year when the file was created